tests/snappi_tests/dataplane/test_latency.py

```diff
@@ -0,0 +1,256 @@
+from tests.snappi_tests.dataplane.imports import *  # noqa F403
+sys.path.append('../test_reporting/telemetry')
+from snappi_tests.dataplane.files.helper import *  # noqa F403
+from metric_definitions import *   # noqa E402
+from reporter_factory import TelemetryReporterFactory  # noqa F403
+from metrics import GaugeMetric  # noqa E402
+
+common_labels = [
```

Comment:
not required to add common labels, likely we can get all information from outside.